WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ProductDiscountView.swift

```diff
@@ -0,0 +1,134 @@
+import SwiftUI
+import Yosemite
+
+struct ProductDiscountView: View {
+    private let imageURL: URL?
+    private let name: String
+    private let stockLabel: String
+    private let productRowViewModel: ProductRowViewModel
+
+    private let minusSign: String = NumberFormatter().minusSign
+
+    @Environment(\.presentationMode) var presentation
+
+    @ObservedObject private var discountViewModel: FeeOrDiscountLineDetailsViewModel
+
+    init(imageURL: URL?,
+         name: String,
+         stockLabel: String,
+         productRowViewModel: ProductRowViewModel,
+         discountViewModel: FeeOrDiscountLineDetailsViewModel) {
+        self.imageURL = imageURL
+        self.name = name
+        self.stockLabel = stockLabel
+        self.productRowViewModel = productRowViewModel
+        self.discountViewModel = discountViewModel
+    }
+
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                HStack(alignment: .center, spacing: Layout.spacing) {
+                    ProductImageThumbnail(productImageURL: imageURL,
+                                          productImageSize: Layout.productImageSize,
+                                          scale: 1,
+                                          productImageCornerRadius: Layout.frameCornerRadius,
+                                          foregroundColor: Color(UIColor.listSmallIcon))
+                    VStack {
+                        Text(name)
+                        CollapsibleProductCardPriceSummary(viewModel: productRowViewModel)
+                    }
+                }
+                .padding()
+                .overlay {
+                    RoundedRectangle(cornerRadius: Layout.frameCornerRadius)
+                        .inset(by: 0.25)
```

Comment:
Another one for the Layout enum

Reply:
ea5b5b3caad410eda9e06624f5b77458d44dfdd5